README.md

```diff
@@ -96,17 +90,31 @@ In your web browser, open the link [http://localhost:3000/](http://localhost:300
 # Running applications with Docker
 1. Install Docker if you haven't already
 2. Open Docker Desktop, you cannot run containers or build images, if docker engine is not running
-2. Navigate project root
+2. Now in terminal, navigate to project root
+3. Run `docker compose up --build`, for the first build it may take a while, after build the application will be ran, you can access the application through the `localhost:3000`.
+5. You can stop containers with ctrl+c or using the Docker app
+
+Common Docker commands:
+- If you want to start the application again, run `docker compose up`
+- If you want to just build images, run `docker compose build`
+- If you want to build and run containers, run `docker compose up --build`
+- If you want to build only one service, run `docker compose build <service-name>`, for example `docker compose build backend`
 
 Note: you don't have to initialize submodule to run docker, dockerfile will set up the submodules for you while building the image.
 
-3. Run `docker compose up --build`, for the first build it may take a while, after build the application will be ran, you can access the application through the `localhost:3000`.
-5. You can stop contianers with ctrl+c or using the Docker app
-6. If you want to start the application again, run `docker compose up`
-7. If you want to just build images, run `docker compose build`
-8. If you want to build and run containers, run `docker compose up --build`
 
-Note: If you want to build only one service, use `docker compose build <service-name>`, for example for backend it will be `docker compose build backend`.
+## Working with database
+
+Sometimes you want to run the application without containers, allowing you to develop things quickly. The **Running the iNatAtor Application** section explains how to run the application locally.
+
+You can verify connections are working by going to `localhost:8000/health` and `localhost:8000/echo_env`. Note that these paths are included right now to assist with rapid development, will be removed for production.
+
+Note: The codebase is getting bigger, therefore add database related code in `src/backend/app/db`, then make proper API routes in main, if it gets too big, we can resort to using API routers from fastapi.
+
+Note: There are two environment files (.env and .docker.env) because the database url for local development and docker environments are separate.
```

Comment:
I think this will introduce a lot of overhead and complication, sqlite might be good for quick development and we used to implement things quickly but we decided that postgres would be what we use in any case. @ginic your thoughts on this?

Reply:
Running postgres is not that difficult on docker, and this request is kind of expanding the scope of the ticket

Reply:
I still don't like the idea of ​​running a postgresql container for local development.
if you just add 2 lines of code to database.py you can:
1. don't start docker at all
2. do not create any .env files locally

Perhaps if we decide to save predictions in the form of coordinates, then sklight will not suit us. but we don't do that yet...

```python
DB_URL = os.getenv("DATABASE_URL")
if not DB_URL:
    DB_URL = "sqlite:///./src/backend/db.sqlite3"
```

Reply:
yes, it should work. But now we have to not forget to create  `.env`  file to put this link there)

Reply:
Sounds good! I will add a suggestion so that information is in the README